app/scripts/directives/realtimestring.js

```diff
@@ -204,9 +204,17 @@ angular.module('oncokbApp')
                         $scope.diffHTML = mainUtils.calculateDiff($scope.data[$scope.key + '_review'].lastReviewed, $scope.data[$scope.key]);
                     }
                 };
-                $scope.toggleCheckbox = function () {
-                    if ($scope.data[$scope.key] === $scope.pureContent.text && $scope.data[$scope.key] !== '') {
-                        $scope.pureContent.text = '';
+                $scope.toggleCheckbox = function (e) {
+                    if ($scope.t === 'radio') {
+                        if ($scope.data[$scope.key] === e.target.value) {
+                            $scope.pureContent.text = '';
```

Comment:
For any value does not match the `$scope.data[$scope.key]`, pureContent.text will never get assigned. What's the use case of that? Should that be handled?

Reply:
We don't need to handle it. `$scope.data[$scope.key] === e.target.value` means the checkbox is already checked and we want to uncheck it. Otherwise, `$scope.pureContent.text` is equal to selected checkbox.value.

Reply:
Oh I see. Here is where you set the value to empty when user double clicks on the radio.